core/src/main/kotlin/cache/data/AutoModeration.kt

```diff
@@ -0,0 +1,90 @@
+package dev.kord.core.cache.data
+
+import dev.kord.common.entity.*
+import dev.kord.common.entity.optional.Optional
+import dev.kord.common.entity.optional.OptionalSnowflake
+import dev.kord.common.entity.optional.map
+import dev.kord.common.serialization.DurationInSeconds
+import kotlinx.serialization.Serializable
+
+@Serializable
+public data class AutoModerationRuleData(
```

Comment:
What do I have to set up so this can be cached? Rules get updates via Gateway, so I think they should be cached.

Reply:
Resolved in https://github.com/kordlib/kord/pull/647/commits/bd864b6923c6530fa72feea1783f8ddb7f1b176c, thanks Hope :)